internal/github/contributions.go

```diff
@@ -129,9 +208,19 @@ func (c LongestContributionStreak) String() string {
 }
 
 func (g *GithubService) GetLongestContributionStreakByUsername(ctx context.Context, username string) (*LongestContributionStreak, error) {
+	year, err := g.GetFirstContributionYearByUsername(ctx, username)
+	if err != nil {
+		return nil, errors.Wrap(err, "get contributions by username")
```

Comment:
can we update this comment to provide correct context
"get first contribution year by username"